client/components/Syncs/SyncsList.js

```diff
@@ -16,7 +16,7 @@ class SyncListContainer extends PureComponent {
   }
   render() {
     let { apiURL } = this.props;
-    apiURL = apiURL.replace(/https?:\/\//g, "ws://");
+    apiURL = apiURL.replace("https://", "wss://").replace("http://", "ws://");
```

Comment:
You could use regex-like syntax here to have 1 replace() only.
```
apiUrl.replace(/^http(s)?:/, "ws$1:")

Reply:
![image](https://user-images.githubusercontent.com/5485824/48142579-52224f80-e2ad-11e8-98b4-f8806560e3df.png)